### DISCUSSION TITLE
Custom list box control

### DISCUSSION THREAD

**Discussion by L-u1** [magiblot/tvision#54] [Q&A] [answered]
Thank you for the library!

It is beautiful.
Could you please prompt - how do I create custom ListBox with header and columns like this?

| ID | Name      | Last Name   | ...    |

| 1  | John         | Smith          | ...    |

| 2  | Joan         | of Arc          | ...    |

Thank you

**Comment by magiblot** [comment, marked as answer]
Hi!

I think there is no default view which does this.

Fortunately I found an old spreadsheet example in http://www.sigala.it/sergio/tvision/resources.html:

![Screenshot_20210406_120242](https://user-images.githubusercontent.com/20713561/113694830-8691d400-96d0-11eb-9cd2-3b3313700087.png)

Although it is not perfect, it can be a good starting point.
You can download it here: [spread.zip](https://github.com/magiblot/tvision/files/6263839/spread.zip)

`sprdview.cpp` contains the class implementations and `spread.cpp` contains the `main` function.

Cheers.

**Reply by L-u1** [reply]
Thank you for the idea, 
 thats the exactly the way I would like to start -> "digging into".

P.S. this *.it is the website I'm hanging whole last week :-D
